From ba34b61685c597a4de852d8c0f2120b8cbfb51d5 Mon Sep 17 00:00:00 2001
From: Michael W <baddaemon87@gmail.com>
Date: Thu, 21 Jun 2018 20:44:02 +0200
Subject: [PATCH 4/9] LineageParts: Fix brightness section

* Devices with multicolor don't automatically allow adjustment of
  brightness
* Adjust the condition so it's removing the section when any of the two
  (multicolor, adjustable brightness) is false

Change-Id: Ib5dffe18c911e737a38d276876e41ab7fc216c46
---
 .../notificationlight/NotificationLightSettings.java            | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/org/lineageos/lineageparts/notificationlight/NotificationLightSettings.java b/src/org/lineageos/lineageparts/notificationlight/NotificationLightSettings.java
index 4f48580..de29819 100644
--- a/src/org/lineageos/lineageparts/notificationlight/NotificationLightSettings.java
+++ b/src/org/lineageos/lineageparts/notificationlight/NotificationLightSettings.java
@@ -149,7 +149,7 @@ public class NotificationLightSettings extends SettingsPreferenceFragment implem
         mScreenOnLightsPref.setOnPreferenceChangeListener(this);
         mCustomEnabledPref = (LineageSystemSettingSwitchPreference)
                 findPreference(LineageSettings.System.NOTIFICATION_LIGHT_PULSE_CUSTOM_ENABLE);
-        if (!mMultiColorLed && !mHALAdjustableBrightness) {
+        if (!mMultiColorLed || !mHALAdjustableBrightness) {
             removePreference(BRIGHTNESS_SECTION);
         }
         if (!mLedCanPulse && !mMultiColorLed) {
-- 
2.17.1

